### PR TITLE
Address new lgtm.com alerts for PR review

### PR DESCRIPTION
**Pull request by hoffmang9**
These need to be reviewed as some effect functionality/logic.

### CODE REVIEW

File: src/consensus/blockchain.py [hunk 1]
```diff
@@ -345,7 +345,7 @@ async def get_sp_and_ip_sub_slots(
         if block is None:
             return None
         is_overflow = self.sub_blocks[block.header_hash].overflow
-        if block is None:
+        if is_overflow is None:
```
Comment (mariano54) [inline src/consensus/blockchain.py:348]: This is incorrect. we should remove the if statment altogether. Doing this in the other branch

File: src/wallet/wallet_blockchain.py [hunk 2]
```diff
@@ -348,7 +348,7 @@ async def get_sp_and_ip_sub_slots(
         if block is None:
             return None
         is_overflow = self.sub_blocks[block.header_hash].overflow
-        if block is None:
+        if is_overflow is None:
```
Comment (mariano54) [inline src/wallet/wallet_blockchain.py:351]: Same thing here

File: src/wallet/wallet_node.py [hunk 3]
```diff
@@ -349,9 +349,10 @@ def has_full_node(self) -> bool:
 
     async def new_peak(self, peak: wallet_protocol.NewPeak, peer: WSChiaConnection):
         assert self.wallet_state_manager is not None
-        curr_peak = self.wallet_state_manager.blockchain.get_peak()
```
Comment (mariano54) [inline src/wallet/wallet_node.py:352]: Why are we removing this code?

**Review by mariano54 — CHANGES_REQUESTED** [review]

File: src/wallet/wallet_node.py (hunk 3 above)
Comment (hoffmang9) [reply src/wallet/wallet_node.py:352]: `curr_peak.weight > curr_peak.weight` That's impossible to be true. I think you mean to compare it to something else but not completely clear from context what that something else is.

Comment (hoffmang9) [reply src/wallet/wallet_node.py:352]: and if you don't test curr_peak then it's an assignment without use.

Comment (mariano54) [reply src/wallet/wallet_node.py:352]: oh yeah, it should be peak.weight > curr_peak.weight @Yostra

**Pull request closed without merging**